.ci/containers/downstream-builder/Dockerfile

```diff
@@ -9,8 +9,7 @@ RUN go get github.com/github/hub
 RUN ssh-keyscan github.com >> /known_hosts
 RUN echo "UserKnownHostsFile /known_hosts" >> /etc/ssh/ssh_config
 
-ENV GOFLAGS "-mod=vendor"
-ENV GO111MODULE "off"
+ENV GO111MODULE "on"
```

Comment:
Historically this has caused `goimports` to hang indefinitely and we don't know why.  It might be the vendor directory, but it also might not be!  We'll have to test it to find out.  Do you have a branch of terraform that's already had the vendor directory removed so we can check?

Reply:
Yup! They're `megan_rm_vendoring` in both downstreams as well.